assets/src/story-embed-block/block/edit.js

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useState, useEffect } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+import { __ } from '@wordpress/i18n';
+import { ResizableBox } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import EmbedControls from './embedControls';
+import EmbedLoadinng from './embedLoading';
+import EmbedPlaceholder from './embedPlaceholder';
+import StoryPlayer from './storyPlayer';
+import { icon } from './index.js';
+import './edit.css';
+
+const MIN_SIZE = 20;
+
+function StoryEmbedEdit({ attributes, setAttributes, className, isSelected }) {
+  const {
+    url: outerURL,
+    width = 360,
+    height = 600,
+    align = 'none',
+    poster,
+    title,
+  } = attributes;
+
+  const [editingURL, setEditingURL] = useState(false);
+  const [localURL, setLocalURL] = useState(outerURL);
+  const [isFetchingData, setIsFetchingData] = useState(false);
+  const [storyData, setStoryData] = useState({});
+  const [cannotEmbed, setCannotEmbed] = useState('');
```

Comment:
this default to `false`

Reply:
Oh, an oversight from when I tried something else today. Good catch!

Reply:
<img width="763" alt="Screenshot 2020-05-11 at 14 59 32" src="https://user-images.githubusercontent.com/237508/81574564-e3401e00-939d-11ea-92fd-51428c7accad.png">

I think it will fix a warning that I am seeing.